Directory.Build.targets

```diff
@@ -159,6 +160,17 @@ We need to run a unity build to restore the test packages and for that reason we
 For that reason, Exec tasks to unity all ignore error. That means that subsequent tasks must exist to validate the command (i.e: parse logs, etc).
 Related: https://forum.unity.com/threads/6572-debugger-agent-unable-to-listen-on-27.500387/  -->
 
+  <!-- Setup Sentry options on Unity UI: dotnet msbuild /t:UnityConfigureSentryOptions src/Sentry.Unity -->
+  <Target Name="UnityConfigureSentryOptions"
+          Condition="'$(MSBuildProjectName)' == 'Sentry.Unity'">
+    <Error Condition="$(UnityRoot) == ''" Text="Couldn't find Unity."></Error>
+
+    <Message Importance="High" Text="Configuring Sentry options."></Message>
+
+    <Exec Command="&quot;$(UnityExec)&quot; -quit -batchmode -nographics -logFile - -projectPath $(UnitySampleProjectPath) -executeMethod Sentry.Unity.Editor.Batch.ConfigureOptions -sentryOptions.Dsn $(TestDsn)"
```

Comment:
Because of `IgnoreStandardErrorWarningFormat="true"`:
If Unity fails for whatever reason, msbuild will ignore the error and proceed.

Is that what we want? We did this in other places here because Unity was always failing with some random, unrelated issue. But in those cases we had an extra step verifying the invocation was successful. For example:

https://github.com/getsentry/sentry-unity/blob/8123a81ff36cbb2c3bffed51b53040a969aaf71f/Directory.Build.targets#L196-L198

Reply:
Removing IgnoreStandardErrorWarningFormat game me the following error:
````
/Users/runner/work/sentry-unity/sentry-unity/Directory.Build.targets(170,5): error MSB3073: The command ""/Applications/Unity/Hub/Editor/2019.4.33f1/Unity.app//Contents/MacOS/Unity" -quit -batchmode -nographics -logFile - -projectPath /Users/runner/work/sentry-unity/sentry-unity/src/Sentry.Unity/../../samples/unity-of-bugs/ -executeMethod Sentry.Unity.Editor.SentryEditorWindowInstrumentation.ConfigureOptions -sentryOptions.Dsn " exited with code -1. [/Users/runner/work/sentry-unity/sentry-unity/src/Sentry.Unity/Sentry.Unity.csproj]
Error: Process completed with exit code 1.
```